### ISSUE TITLE
Confusing Input Parametes for PURPLE

### ISSUE THREAD

**Issue by DarioS** [hartwigmedical/hmftools#27]
Purple has options like `-ref_sample` and `-tumor_sample` but none similar to `-reference_bam` and ` -tumor_bam` like AMBER has. I am confused why there are no input files specified in the PURPLE example - only labels for the pair of mysterious samples. My impression is that PURPLE contains both AMBER and COBALT in its JAR, so AMBER and COBALT do not need to be run separately before PURPLE is run. So, PURPLE should also accept BAM file paths. Is it correct? The README file gives me that impression.

**Comment by p-priestley** [comment]
Thanks again for the feedback.  We have generally worked with users actively to help them setup.   We will make sure the documentation is complete with examples in coming days.

**Comment by jonbaber** [comment]
PURPLE depends on the output from AMBER and COBALT. The PURPLE jar does contain AMBER and COBALT within it, but they still need to be run independently. I have updated the readme to try and make this clearer in AMBER, COBALT and PURPLE.

I agree that ref_sample and tumor_sample are confusing names. I have released a new version of purple [(2.21)](https://github.com/hartwigmedical/hmftools/releases/tag/purple-v2-21) that has more consistent naming, ie "tumor" and "reference".  These are the names of the samples in the BAMS / VCFs. They should be the same as what was used in AMBER and COBALT.

I'll leave this ticked open for now in case you have any further feedback after these changes.

Jon
